src/actions.sol

```diff
@@ -50,30 +64,34 @@ contract Actions {
         borrowWithdraw(shelf, loan, amount, usr);
     }
 
-    function issueLockBorrowWithdraw(ShelfLike shelf, address registry, uint token, uint amount, address usr) public {
-        uint loan = shelf.issue(registry, token);
+    function transferIssueLockBorrowWithdraw(ShelfLike shelf, NFTLike registry, uint token, uint amount, address usr) public {
```

Comment:
We could call the `borrow` in the shelf `deposit` and rename this entire action to `borrow`.

Reply:
yes, I was thinking about that. I guess too late because of the audit

Reply:
i agree, it's confusing to explain the difference between `withdraw` and `borrow`. but at the very least, maybe we could rename these methods to something a bit shorter? like, maybe just `withdraw`, and then add comments on what is going on?

Reply:
its a combination of all the functions that are executed in the shelf... considering that we have different combinations, I like this pattern.

Reply:
just also thinking about tinlake.js 😂

Reply:
`issueWithdraw` v just `withdraw`?

Reply:
none of this is really user frontend, so the naming is fine I think :)